src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -106,16 +131,57 @@ private void initComponents() {
         panelFooter = getPanelFooter();
         getContentPane().add(panelFooter, BorderLayout.SOUTH);
 
-        for (int i = 0; i < pColumwidth.length; i++) {
-            jTableInventory.getColumnModel().getColumn(i).setMinWidth(pColumwidth[i]);
-        }
+        ajustWidth();
 
         addWindowListener(new WindowAdapter() {
             public void windowClosing(WindowEvent e) {
                 dispose();
             }
         });
-        // pack();
+		pagesCombo.setEditable(true);
+		previous.setEnabled(false);
+		next.setEnabled(false);
+		next.addActionListener(actionEvent -> {
+			if (!previous.isEnabled()) {
+				previous.setEnabled(true);
+			}
+			startIndex += PAGE_SIZE;
+			jTableInventory.setModel(new InventoryBrowsingModel(startIndex, PAGE_SIZE));
+			if ((startIndex + PAGE_SIZE) > totalRows) {
```

Comment:
```suggestion
			if (startIndex + PAGE_SIZE > totalRows) {
```
Unnecessary parentheses